src/Middleware/RequestLimiter/sample/Startup.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.ResourceLimits;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.RequestLimiter;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace RateLimiterSample
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllersWithViews();
+            services.AddSingleton(new IPAggregatedRateLimiter(2, 2));
+            services.AddSingleton(new TokenBucketRateLimiter(2, 2));
+
+            services.AddRequestLimiter(options =>
+            {
+                options.SetDefaultPolicy(new ConcurrencyLimiter(new ConcurrencyLimiterOptions { ResourceLimit = 100 }));
+                options.AddPolicy("ipPolicy", policy =>
+                {
+                    // Add instance
+                    policy.AddAggregatedLimiter(new IPAggregatedRateLimiter(2, 2));
+                });
+                options.AddPolicy("rate", policy =>
+                {
+                    // Add from DI
+                    policy.AddLimiter<TokenBucketRateLimiter>();
+                });
+            });
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<Startup> logger)
+        {
+            app.UseRouting();
+
+            app.UseRateLimiter();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapGet("/defaultPolicy", async context =>
+                {
+                    await Task.Delay(5000);
+                    await context.Response.WriteAsync("Default!");
+                }).EnforceLimit();
```

Comment:
`EnforceLimit` gives no context here for what kind of limit.
`LimitRequests`?
`LimitConcurrency`?
`LimitRequestRate`? as opposed to body rate

Reply:
This one enforces the default limit by adding a blank attribute.

Reply:
Yeah, we want to distinguish between this and other limits like request sizes, data rates, etc..